scripts/code-server.sh

```diff
@@ -94,4 +94,4 @@ if [[ ${code_server_settings} && ${code_server_settings-_} ]]; then
 fi
 
 # Force restart tmux session to reload terminal
-killall -9 /home/ubuntu/.strigo/tmux
+killall -9 /home/ubuntu/.strigo/tmux || true
```

Comment:
Je viens de tester. Cela fonctionne mais affiche toujours un message d'erreur. 
Je verrai plutot un truc comme ceci ? 
```
# Force restart tmux session to reload groups
FILE=/home/ubuntu/.strigo/tmux
if test -f "$FILE"; then
        killall -9 /home/ubuntu/.strigo/tmux || true
fi
```

Reply:
Ah yes, the command still display an error!